packages/components/src/components/domainproperties/models.ts

```diff
@@ -1137,7 +1143,7 @@ export function resolveAvailableTypes(
     showFilePropertyType?: boolean
 ): List<PropDescType> {
     // field has not been saved -- display all property types allowed by app
```

Comment:
Should update or add to this comment about why the addition of the wrappedColumnName check